contracts/Marketplace.sol

```diff
@@ -212,10 +211,6 @@ contract Marketplace is Collateral, Proofs, StateRetrieval {
     require(token.transfer(msg.sender, amount), "Withdraw failed");
   }
 
-  function getHost(SlotId slotId) public view returns (address) {
```

Comment:
I have moved this function into the `TestCollateral`

Reply:
Well, we actually need this in the Codex client, so moving it to TestCollateral won't work :smile: The sales module in nim-codex relies on being able to see which host filled a slot.

Reply:
Oh, damn 🙈 I will revert it then.